jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -59,46 +78,116 @@
 	@Resource
 	protected javax.ejb.TimerService timerService;
 
+    @Resource
+    protected javax.ejb.SessionContext sessionContext;
+
+    @Resource
+    protected TransactionSynchronizationRegistry tsr;
```

Comment:
this seems to be not used

Reply:
done

Reply:
I still can’t find any use of the tsr... maybe it should be removed if it is not used?

Reply:
I can confirm this is not used at all.